tests/python/openassetio/hostApi/test_manager.py

```diff
@@ -538,8 +558,9 @@ def test_wraps_the_corresponding_method_of_the_held_interface(
 
 
 class Test_Manager_preflight:
-    def test_method_defined_in_python(self):
+    def test_method_defined_in_cpp(self):
```

Comment:
[fix] Think this snuck in to this commit - might need rescuing when dropping when rebasing :)

Reply:
If I understand correctly, I think that's done. See first commit, 97796b12

Reply:
682ac146 is now the first change of this line in the PR